lib/active_fedora/scoping/named.rb

```diff
@@ -13,6 +13,7 @@ module ClassMethods
         #
         #   fruits = Fruit.all
         #   fruits = fruits.where(color: 'red') if options[:red_only]
+        #   fruits = fruits.where(size: ['big', 'little'], join_with: ' OR ') #gets big or little fruit
```

Comment:
@mjgiarlo & @jcoyne: see all the comments below....  I do not think that changing to a OR in this branch is a great idea.